createcoveragereport.sh

```diff
@@ -37,4 +58,8 @@ $REPORTGENERATOR \
    -targetdir:coverage/report \
    -verbosity:Error
 
-# TODO(benwu): upload coverage/report to public blob storage
+choco install codecov
```

Comment:
This should be conditional - see earlier comment.
Basically, we still want to be able to run createcoverage.sh locally without uploading, but we want it to have an optional final step.